client/src/components/UserDetails.tsx

```diff
@@ -13,20 +13,13 @@ interface Props {
 const UserDetails = (props: Props) => (
   <>
     <div className="user-card__column-1">
-      <h3 className="heading">User name</h3>
+      <h3 className="heading">Full name</h3>
```

Comment:
yes, they are interchangeable.  Username has been used all along, but I want to signify to the end user that this is their full name and not just a username.  This is now being used to check their identity with identity/get

Reply:
I seee. I can see this being a bit confusing for clients who are referencing the pattern code. I wonder if it makes sense to add a separate field for `Full Name` (separate from username) and this full name field is the one used for the identity check. Wdyt?

Because reading the code where the `username` var is actually the full name (with spaces and such) seems confusing. In other apps, username and full name are typically separate fields.

Reply:
yes, I agree. I will make this change.